CONTRIBUTING.rst

```diff
@@ -55,3 +55,13 @@ to see how that works.
 .. _Read the Docs: https://commcare-hq.readthedocs.io/
 .. _reStructuredText: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html
 .. _autoclass: https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html
+
+
+Updating requirements
+---------------------
+To update requirements edit ``requirements/requirements.in``
```

Comment:
Should we mention which file should be updated since we have different files for general requirements, tests, production and dev?

Reply:
Sure. Would you mind taking a crack at it?

Reply:
https://github.com/dimagi/commcare-hq/pull/26462